src/components/VmActions/ConfirmationModal.js

```diff
@@ -30,7 +30,7 @@ const ConfirmationModal = ({ show, title, confirm, body, onClose, extra, accessi
       <Modal.Footer>
         { extra && <button className='btn btn-info' onClick={() => { extra.onClick(); onClose() }}>{extra.title}</button> }
         <button className={`btn ${confirm ? 'btn-default' : 'btn-info'}`} onClick={onClose}>{msg.cancel()}</button>
-        { confirm && <button className='btn btn-info' onClick={() => { confirm.onClick(); onClose() }}>{confirm.title}</button> }
+        { confirm && <button className='btn btn-danger' onClick={() => { confirm.onClick(); onClose() }}>{confirm.title}</button> }
```

Comment:
`ConfirmationModal` is more general component. It means that this component is used not only to confirm some danger actions, but also to show some information, for example when console SSO logon fails, it asks user if user want open console. So I think it would be better to add additional props to `confirm` which indicates that button should be `danger`.

Reply:
I added an if statement before determine the button type, and if it contain remove message, It  becomes `danger`, otherwize it stay `info`.